### ISSUE TITLE
Code executed in info.py is confusing

### ISSUE THREAD

**Issue by ojnas** [Telecominfraproject/oopt-gnpy#115]
The following code in info.py looks wrong:
```
if __name__ == '__main__':
    si = SpectralInformation(
        Channel(1, 193.95e12, 32e9, 0.15,  # 193.95 THz, 32 Gbaud
            Power(1e-3, 1e-6, 1e-6)),             # 1 mW, 1uW, 1uW
        Channel(1, 195.95e12, 32e9, 0.15,  # 195.95 THz, 32 Gbaud
            Power(1.2e-3, 1e-6, 1e-6)),           # 1.2 mW, 1uW, 1uW
    )
```
I realize this code is not meant to be used anywhere. But I guess the intention is to show an example of how to create a new instance of the SpectralInformation class. In this case it is very confusing since the result is that pref is set to the first Channel and carriers is set to the second Channel.

**Comment by jeanluc-auge** [comment]
@dutc , there is a test in place to check the spectrum information,
https://github.com/Telecominfraproject/oopt-gnpy/blob/a0fac17c0e24c9a54314897d798a9286aec73150/tests/test_amplifier.py#L104
it's just this piece of code that is not used.

**Comment by dutc** [comment]
Fixed in version 1.0.
